index.html

```diff
@@ -305,7 +341,7 @@ <h2>About <br> Myself</h2>
         <!--Get my resume-->
 
 
-        <div class="Action">
+        <div class="Action resume">
           <button>Get my resume</button>
```

Comment:
- [ ] Good job on your design 👍🏽 . Kindly use the appropriate color on all buttons on your project to match the Figma design template you chose 👍🏽 . Your button texts are black all through which is not in the design. Please fix it 👍🏽